Core/Rewriters/SetupResultForRewriter.cs

```diff
@@ -0,0 +1,67 @@
+﻿//  Copyright (c) rubicon IT GmbH
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using System;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using CSharpExtensions = Microsoft.CodeAnalysis.CSharpExtensions;
+using ModelExtensions = Microsoft.CodeAnalysis.ModelExtensions;
+using SyntaxNodeExtensions = Microsoft.CodeAnalysis.SyntaxNodeExtensions;
+
+namespace RhinoMocksToMoqRewriter.Core.Rewriters
+{
+  public class SetupResultForRewriter : RewriterBase
+  {
+    public override SyntaxNode? VisitExpressionStatement (ExpressionStatementSyntax node)
+    {
+      var trackedNodes = SyntaxNodeTrackingExtensions.TrackNodes (node, node.DescendantNodesAndSelf(), CompilationId);
```

Comment:
Don't you only need to track expression statements here?

Reply:
> Don't you only need to track expression statements here?

No I also need to track the `SetupResult` and `For` identifiername and the easiest way to do this is to track every descendant node

Reply:
> ... I also need to ...

Also? shouldn't you only need to track `SimpleMemberAccessExpression` nodes then?